frontend/src/components/common/Resource.tsx

```diff
@@ -331,23 +332,32 @@ export function SectionGrid(props: SectionGridProps) {
 
 export function DataField(props: TextFieldProps) {
   const { label, value, ...other } = props;
+  let language = ((label as string).split('.').pop() as string);
+  if (language !== 'json') {
+    language = 'yaml';
+  }
   return (
-    <TextField
-      label={label}
-      InputProps={{
-        readOnly: true,
-      }}
-      InputLabelProps={{
-        shrink: true,
-        style: {fontSize: '1.3rem'}
-      }}
-      variant="outlined"
-      fullWidth
-      multiline
-      rowsMax="20"
-      value={value}
-      {...other}
-    />
+    <>
+      <TextField
+        label={label}
+        InputLabelProps={{
+          shrink: true,
+          style: {fontSize: '1.3rem'}
+        }}
+        variant="outlined"
+        fullWidth
+        multiline
+        rowsMax="20"
+        {...other}
+      />
```

Comment:
I don't think we want a `TextField` anymore since we're not using it. What we need is an https://material-ui.com/api/input-label/ with the Editor within.
![Screenshot from 2020-09-08 12-44-52](https://user-images.githubusercontent.com/1029635/92472912-02c34800-f1d2-11ea-834e-aca2d8fd8528.png)